pxd_fastpath.c

```diff
@@ -1513,8 +1255,29 @@ void pxd_make_request_fastpath(struct request_queue *q, struct bio *bio)
 		return BLK_QC_RETVAL;
 	}
 
-	tc = get_thread_context(rw);
-	pxd_add_io(tc, head, rw);
+{
+	// If IO suspended, then hang IO onto the suspend wait queue
+	int cpu = get_cpu();
```

Comment:
you can just make a single call to get the local percpu rt?

Reply:
get_cpu ensures preemption is disabled, so context is still correct.